specification/common-types/data-plane/v1/types.json

```diff
@@ -41,18 +41,25 @@
           },
           "description": "The error additional info."
         }
```

Comment:
Might want to remove "additionalInfo" -- not really part of the common response schema.